README.md

```diff
@@ -142,20 +115,20 @@ OverlappingSlider.init({
 
 ## Events
 
-| Name                        | Description | 
-|-----------------------------|-------------|
-| `onPause: (data) => {}`     |             |
-| `onPlay: (data) => {}`      |             |
-| `onChange: (data,el) => {}` |             |
+| Name                        | Description                                                                            | 
+|-----------------------------|----------------------------------------------------------------------------------------|
+| `onPause: (data) => {}`     | Trigger after turning off the autoplay setting.                                        |
+| `onPlay: (data) => {}`      | Trigger after turning on the autoplay setting.                                         |
+| `onChange: (data,el) => {}` | When you call this event, your Overlapping Slider will re-initialize with new options. |
```

Comment:
> Trigger after turning off the autoplay setting.
1. When talking about an event, the verb "fire" would be more accurate than "trigger".
2. I can't find anything in common between `onChange` and `re-initialize with new options`.